src/common/intl/messages.js

```diff
@@ -1441,6 +1443,9 @@ const messages = {
   logNoStackTrace: 'No stack trace',
   logNoParams: 'No params',
   logDelete: 'Delete log',
+  logDeleteSelected: 'Delete selected logs',
+  logDeleteSlectedMessage:
```

Comment:
Typo.

Reply:
These should probably be called `logDeleteMultiple` and `logDeleteMultipleMessage` since they're no related to selected logs.